src/main/java/com/softserveinc/ita/pageobjects/admin/AddingFacultyModal.java

```diff
@@ -0,0 +1,14 @@
+package com.softserveinc.ita.pageobjects.admin;
+
+import static com.codeborne.selenide.Condition.*;
+import static com.codeborne.selenide.Selenide.$x;
+import static java.time.Duration.*;
+
+public class AddingFacultyModal {
```

Comment:
take a look at existing modal objects, maybe there is something common with each other (buttons or titles) and it would be possible to create some parent abstract class with common variables or methods